### DISCUSSION TITLE
Error loading kernel for NVIDIA GeForce RTX 4070 Ti[89].

### DISCUSSION THREAD

**Discussion by X5-452MAX** [NebuTech/NBMiner#972] [General]
[23:15:34] ERROR - Error loading kernel for NVIDIA GeForce RTX 4070 Ti[89].
[23:15:34] WARN - QMutex: destroying locked mutex

what is the problem?

**Comment by rezor555** [comment]
got the same problem xD

**Comment by tomtkr** [comment]
got the same problem

**Comment by AMV007** [comment]
+1 same problem with latest drivers at the latest ubuntu

**Comment by Lawgend** [comment]
Still no reply yet from developer?

**Comment by Stankul11** [comment]
got the same problem

**Comment by Stankul11** [comment]
> [23:15:34] ERROR - Error loading kernel for NVIDIA GeForce RTX 4070 Ti[89]. [23:15:34] WARN - QMutex: destroying locked mutex
> 
> what is the problem?

**Reply by VasTShadowZz** [reply]
got the same problem

**Comment by marcolin1215** [comment]
+1 same problem

**Comment by neotist** [comment]
Still no reply?

**Comment by YawataNoKami** [comment]
+1 :(

**Comment by GhostAndry** [comment]
+1 but with 4060 not ti
```
Error loading kernel for NVIDIA GeForce RTX 4060[89]
```

**Reply by devlopernoor** [reply]
Error loading kernel for NVIDIA GeForce R Error loading kernel for NVIDIA GeForce RTX 4060 Laptop GPU[89].TX 4060 Laptop GPU[89].

**Reply by GhostAndry** [reply]
i have a desktop version

**Comment by wjc2821296948** [comment]
一样+1
